src/NexusMods.App.UI/Windows/MainWindowViewModel.cs

```diff
@@ -135,6 +139,31 @@ private IDisposable ConnectErrors(IServiceProvider provider)
             );
     }
 
+    private static bool ShouldShowError(LogMessage msg)
+    {
+        // Note(sewer): On some Wayland compositors on Linux, there is currently a bug 
+        // where showing tooltips can trigger an exception dialog.
+        //
+        //   System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable)
+        //    ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
+        //      at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
+        //      at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
+        //      at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
+        //      --- End of inner exception stack trace ---
+        //
+        // This needs to be fixed on Avalonia's end, unfortunately, which may take a while
+        // and wouldn't quite cut it for Stardew Valley Beta.
+        //
+        // For now, we opt to ignore this exception for the time being, while either us
+        // or the Avalonia folks come up with a solution. Whichever is first.
+        // ReSharper disable once MergeIntoPattern
+        if (msg.Exception is AggregateException aggregateException 
+            && aggregateException.InnerException is DBusException)
+            return false;
```

Comment:
I don't think we should be excluding all D-Bus exceptions. D-Bus is used for selecting and opening files and links, among other things.

Reply:
Do you have a better suggestion? We can't necessarily filter out by using the exception message; that may be localized; or contain localized parts.

The HRESULT is 0x80131500, which, as far as I can tell would likely be shared by other DBus Services.

The Stack Trace does not have any Avalonia code etc. we don't even know the caller of the issue. Nothing we could reference there either.

We have no hope of getting Avalonia to merge our changes, even if we were to fix it ourselves within the next 4-8 working days.